libs/application/form/src/components/Screen.tsx

```diff
@@ -147,43 +151,75 @@ const Screen: FC<ScreenProps> = ({
               />
             )}
           </Box>
-        </Box>
-        <Box marginTop={[3, 3, 0]}>
-          <Divider weight="regular" />
-          <Box
-            display="flex"
-            flexDirection="row"
-            justifyContent="spaceBetween"
-            paddingTop={[1, 4]}
-            paddingBottom={[1, 5]}
+        </GridColumn>
+        <Box marginTop={[3, 3, 0]} className={styles.buttonContainer}>
+          <GridColumn
+            span={['12/12', '12/12', '7/9', '7/9']}
+            offset={[null, null, '1/9']}
           >
-            <Box display="inlineFlex" padding={2} paddingLeft="none">
-              <Button variant="text" leftIcon="arrowLeft" onClick={goBack}>
-                Til baka
-              </Button>
-            </Box>
-            <Box display="inlineFlex" padding={2} paddingRight="none">
-              {shouldSubmit ? (
-                <Button
-                  loading={loading}
-                  disabled={!canProceed()}
-                  htmlType="submit"
-                >
-                  Submit
+            <Box
```

Comment:
I think very soon we might need to extract the screen footer into a separate component. I am adding logic to it as well, and it might include more buttons than "Continue" on the right. Will address this later